libs/components/src/lib/dial-pad/dial-pad.scss

```
@@ -0,0 +1,40 @@
+@use '../../../../../dist/libs/tokens/scss/tokens.constants' as constants;
+@use '../../../../shared/src/lib/sass/mixins/border-radius' as
+	border-radius-variables;
+
+$inline-size: 230px;
+$gap: 16px;
+
+.base {
+	display: grid;
+	margin: 16px;
+	gap: $gap;
+	inline-size: $inline-size;
+}
+
+.digits {
+	display: grid;
+	gap: $gap;
+	grid-template-columns: repeat(3, 1fr);
+	grid-template-rows: repeat(4, 1fr);
+	inline-size: $inline-size;
+}
+
+.phone-field {
+	--vvd-size-density: 2;
+
+	align-self: flex-start;
+	grid-column: 1 / -1;
+}
+
+.digit-btn {
+	display: flex;
+	flex-direction: column;
+	border: 1px solid var(#{constants.$vvd-color-neutral-300});
+	border-radius: #{border-radius-variables.$border-radius-extra-expanded};
```

```suggestion
	border-radius: #{border-radius-variables.$border-radius-expanded};
 overflow: hidden;
 inline-size: 100%;
```
I would go with this one, and remove the shape-pill from the the digits buttons.

---

@AyalaBu approved this shape, I don't think we can change it